src/operations/blas3/gemm_no_local.hpp

```diff
@@ -298,7 +298,7 @@ class Gemm<input_t, output_t, DoubleBuffer, NbcA, NbcB, ClSize, tile_type,
       for (index_t i = 0; i < item_rows; ++i) {
         if (do_check<need_check_boundary>(check_boundary(
                 dim_m_c_start + i * wg_rows, dim_n_c_start + j * wg_cols))) {
-          reg_res[i * item_rows + j] = beta * C[i * wg_rows];
+          reg_res[j * item_rows + i] = beta * C[i * wg_rows];
```

Comment:
can you adjust the index everywhere to be either  i + j*xxx or j+ i* xxx

Reply:
Fixed!